source/github/repo.go

```diff
@@ -16,6 +17,17 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
```

Comment:
This function is missing its comment

Reply:
Moved text fromMoved "ConfigBackoff defines a function wrapper that checks if yml/yaml file is valid" comment from source.go to here instead.